src/NServiceBus.Transport.SQS/TransportMessage.cs

```diff
@@ -28,9 +28,9 @@ public TransportMessage(OutgoingMessage outgoingMessage, DispatchProperties prop
             }
 
 #if NETFRAMEWORK
-            Body = outgoingMessage.Body.Length != 0 ? Convert.ToBase64String(outgoingMessage.Body.ToArray()) : "empty message";
```

Comment:
If someone has two endpoints today (Sender and Receiver) that are exchanging empty messages, this will break things if the sender is upgraded before the receiver. Is that likely (or even supported)?